.github/dependabot.yml

```diff
@@ -0,0 +1,26 @@
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    commit-message:
+      prefix: "bot-chore:"
+    open-pull-requests-limit: 10
+    ignore:
+      - dependency-name: "*"
+    security-updates: true
```

Comment:
Couldn't find this top level setting in the [dependabot docs](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file), sure that it is correct?